silkrpc/context_pool.cpp

```diff
@@ -48,7 +49,8 @@ ContextPool::ContextPool(std::size_t pool_size, ChannelFactory create_channel) :
         auto grpc_runner = std::make_unique<CompletionRunner>(*grpc_queue, *io_context);
         auto database = std::make_unique<ethdb::kv::RemoteDatabase<>>(*io_context, grpc_channel, grpc_queue.get()); // TODO(canepat): move elsewhere
         auto backend = std::make_unique<ethbackend::BackEnd>(*io_context, grpc_channel, grpc_queue.get()); // TODO(canepat): move elsewhere
-        contexts_.push_back({io_context, std::move(grpc_queue), std::move(grpc_runner), std::move(database), std::move(backend)});
+        auto tx_pool = std::make_unique<silkrpc::txpool::TransactionPool>(*io_context, grpc_channel, grpc_queue.get()); // TODO(canepat): move elsewhere
```

Comment:
Use `txpool::TransactionPool` instead of `silkrpc::txpool::TransactionPool`, adding the namespace `silkrpc::` is redundant here